src/data/faq_de.json

```diff
@@ -54,7 +54,7 @@
                     "title" : "[Apple/iOS]: Zu wenig Speicherplatz",
                     "anchor": "memory",
                     "textblock": [
-                        "Diese Fehlermeldung kommt aus dem Betriebssystem. Apple ist informiert und arbeitet an einer Behebung des Fehlers. Die Corona-Warn-App benötigt nur ca. 20 MB Speicherplatz auf dem Handy. Die Größe kann sich durch eventuelle Updates laufend verändern (wenn auch nur minimal). Zusätzlich fallen weitere Speicherkapazitäten durch die von der App zwischengespeicherten Daten an."
```

Comment:
@kkopper What are the changes here?

Reply:
I haven't changed anything here myself, this change was induced by GitHub. As *Größe*, *verändern*, *zusätzlich* and *Speicherkapazitäten* are highlighted, I suspect this is somewhat related to the umlaute.